hack/update-crdb-versions.sh

```diff
@@ -47,8 +51,14 @@ cat > crdb-versions.yaml << EOF
 
 CrdbVersions:
 EOF
+}
+
+dump_versions() {
+  local nameFilter=".data[] .repositories[] .tags[] .name"
+  # Skip unsupported versions and the latest tag
+  for version in $(curl -s $URL | jq -r "${nameFilter}" | grep -v ^v19 | grep -v ^v21.1.8$ | grep -v latest | grep -v ubi$ | sort --version-sort); do
```

Comment:
This is a pretty complicated command that can fail a lot of ways.  Including jq is not installed.  Should we break this into more steps and check that the data is returned is valid?

Reply:
I agree about breaking this up. However, this is in place now. Meaning jq is installed in the CI environment where the PR bot runs.

I was talking with @davidwding earlier about converting some of these critical bash scripts into some nice, testable, go apps. A problem for another PR though.

Reply:
I agree, I think we can file an issue and ship this

Reply:
Issue logged https://github.com/cockroachdb/cockroach-operator/issues/738

Reply:
@chrislovecnm you good with this now?